www/source/docs/pattern-library.html.md.erb

```diff
@@ -181,3 +181,115 @@ spec:
         requests:
           storage: 10Gi
 ```
+
+## <a name="hab-pkg-download" id="hab-pkg-download" data-magellan-target="hab-pkg-download" type="anchor">hab pkg download Patterns</a>
+
+The `hab pkg download` command can be used to download individual
+packages (along with their dependencies and keys) from Builder,
+without installing them. This allows you do more easily transfer
+packages from one Builder instance to another, or to take a selective
+snapshot of particular packages.
+
+While you can download packages one-at-a-time, it can be more
+convenient to use a file to specify your packages. Two formats are
+recognized: plain text and TOML.
+
+### Plain Text Download Descriptors
+
+The simplest thing you can do is create a plain text file with a
+package identifier on each line, like so:
+
+```text
+# These are the packages needed to run a Supervisor
+core/hab-launcher
+core/hab
+core/hab-sup
+```
+
+Each line is a valid package identifier. You can also add comments using `#`.
+
+To download these packages (and their dependencies), save that to a
+file named `supervisor.txt` and run:
+
+```
+hab pkg download --file=supervisor.txt
+```
+
+This will download the packages into your existing Habitat cache
+directory. Alternatively, you can specify a directory using the
+`--download-directory` option.
+
+(You can also specify `--channel` and `--target` to further control
+which specific packages you download; run `hab pkg download --help`
+for more).
+
+### TOML Download Descriptors
+
+Plain text is fine for simple cases, but has drawbacks. For instance,
+all packages will come from the same channel, and will be for the same
```

Comment:
can omit comma

Reply:
Fixed in https://github.com/habitat-sh/habitat/pull/7457/commits/6206e45e9519bfaa582ea377b25e3aff2c9da352